portfolio/src/main/webapp/index.html

```diff
@@ -6,13 +6,27 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body style="font-family:Georgia;background-color:#937691;">  <!-- Light purple-->
```

Comment:
can you add your styles to a css file, rather than inlining them? Inline CSS is much much harder to maintain and understand.